global.yaml

```diff
@@ -98,6 +98,18 @@ global:
       enabled: false
       host: # example: accesslog-service.istio-system
       port: # example: 15000
+      tlsSettings:
```

Comment:
How about replacing this with 'envoyAccessLogService' - and a link to wherever this is documented for the structure.